### PR TITLE
remove sbom-json-check from konflux pipelines

### PR DESCRIPTION
**Pull request by dem4gus**
This change is for the sbom-json-check migration to 0.2, from #281

### CODE REVIEW

**Comment by dem4gus** [comment]
/hold

Will need to be rebased after #281 is merged.

**Review by sam-nguyen7 — APPROVED** [review]
/lgtm

**Pull request merged**
